applications/test/GalleryTemplateTest.scala

```diff
@@ -55,10 +55,11 @@ import scala.jdk.CollectionConverters._
 
   it should "select the largest main picture for the opengraph image when FacebookShareUseTrailPicFirstSwitch is OFF" in {
     FacebookShareUseTrailPicFirstSwitch.switchOff()
-    goTo("/lifeandstyle/gallery/2014/nov/24/flying-dogs-in-pictures") { browser =>
+    goTo("/community/gallery/2024/jan/17/a-salty-sea-dog-and-a-vintage-wall-readers-best-photos") { browser =>
```

Comment:
I will revert this commit or try with a different article. Selenium is not loading this page in CI. The test is flaky locally, failing with the same error, although most times is passing.